src/css/custom.css

```diff
@@ -37,6 +37,10 @@ body {
   max-width: 1440px;
 }
 
+.ams-grid_inner {
```

Comment:
Is there a reason this is not in the `ams-overrides.css` file?

Reply:
Nope, was a little unclear to me that it should be in the override file. I thought it is a new class and therefor could be in the custom file. Moved it though.